localgraphclustering/tests/test_notebooks.py

```diff
@@ -29,4 +30,76 @@ def _exec_notebook(notebook_filename):
     ep.preprocess(nb, {'metadata': {'path': 'notebooks/'}})
 
 def test_examples():
-	_exec_notebook('notebooks/examples.ipynb')
+    _exec_notebook('notebooks/examples.ipynb')
+"""
```

Comment:
We should add a link to where this code came from!

Reply:
Ok, working on it. I also in the middle of adding more tests to get the coverage higher than 90% hopefully.